core/consensus/grandpa/has_change.hpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CONSENSUS_GRANDPA_HAS_CHANGE_HPP
+#define KAGOME_CONSENSUS_GRANDPA_HAS_CHANGE_HPP
+
+#include "primitives/block_header.hpp"
+
+namespace kagome::consensus::grandpa {
+  struct HasChange {
+    HasChange(const primitives::BlockHeader &block) {
```

Comment:
Why is this not a function, by the way?

Reply:
Function would need this return type anyway.  
This function returns if there were scheduled and/or forced change.  
Warp sync needs this to separate forced changes from scheduled.